### PR TITLE
Support multiple cohorts in local evaluation

### PR DESCRIPTION
**Pull request by zaynetro**
> 3. There's more than one cohort in the feature flag definition.

This restriction is annoying. Copying what [NodeJS SDK does](https://github.com/PostHog/posthog-js-lite/pull/85).

Ref: https://posthog.com/docs/feature-flags/local-evaluation

PS Sorry I couldn't reopen my old PR somehow :/

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-go 1.15
+go 1.18
 
 require (
 	github.com/google/uuid v1.3.0
 	github.com/urfave/cli v1.22.5
 )
+
+require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d // indirect
```
Comment (neilkakkar) [inline go.mod:11]: these are a bit sus, what are they used for? 🤔

Comment (zaynetro) [reply go.mod:11]: I just ran `go mod tidy`. This seems to be a dependency of [`github.com/urfave/cli`](https://github.com/urfave/cli/blob/v1.22.5/go.mod#L7).

**Review by neilkakkar — APPROVED** [review]

**Comment by zaynetro** [comment]
Hi! Do you plan on merging this soonish? Is there something I can help with?

**Comment by neilkakkar** [comment]
Sorry was off sick with a bad cold, didn't write a proper note on the approval. I'm just going to test this version with cohorts with a real project, and merge when all still looks good!

Thanks again for doing this :)

**Comment by neilkakkar** [comment]
All good here :) - I found a separate issue where non-integer rollout percentages bork unmarshalling, but nothing related to this 👍

**Comment by neilkakkar** [comment]
Thanks @zaynetro for all your recent contributions. I'd love to send you a posthog merch code, could you send me an email at neil @ posthog . com ?

**Pull request merged**